packages/graphback-core/src/runtime/CRUDService.ts

```diff
@@ -1,4 +1,4 @@
-import * as DataLoader from "dataloader";
+import DataLoader from "dataloader";
```

Comment:
Even though the tsconfig is adjusted, I am getting:

```
src/SchemaCRUDPlugin.ts(724,39): error TS2351: This expression is not constructable.
  Type 'typeof DataLoader' has no construct signatures.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This is after running `yarn unlock && yarn clean && rm -rf node_modules && yarn && yarn build` to ensure a clean repo.

Reply:
Did we need to change the way we imported everything because of changes made to the `tsconfig.json` file?

Reply:
Lets use ts file